tests/dependency_analyzer/cyclonedx/resources/bom_no_group.json

```diff
@@ -63,16 +63,6 @@
       ],
       "type" : "library",
       "bom-ref" : "pkg:maven/joda-time/joda-time@2.6?type=jar"
-    },
```

Comment:
I wonder what is the issue of this component to be removed from this BOM file. This same question applies to changes in `tests/dependency_analyzer/cyclonedx/resources/bom_no_version.json`